clang/test/Driver/aarch64-mcpu-native.c

```diff
@@ -0,0 +1,137 @@
+// RUN: export LLVM_CPUINFO=%S/Inputs/cpunative/neoverse-v2
+// RUN: %clang --target=aarch64 --print-enabled-extensions -mcpu=native | FileCheck --strict-whitespace --check-prefix=CHECK-FEAT-NV2 --implicit-check-not=FEAT_ %s
+
```

Comment:
Will this test need `// REQUIRES: aarch64-registered-target`?

Reply:
thanks, you're right, added it